documentation/tutorial_setupProduction.md

```diff
@@ -0,0 +1,54 @@
+# Tutorial: Setting Up a Production Instance
+
+### Overview
+
+This tutorial demonstrates how to set up a production ready environment for a React PWA Reference Storefront.
```

Comment:
This tutorial demonstrates setting up a production ready environment for a React PWA Reference Storefront with an example.